app/views/admin/settings/index.html.erb

```diff
@@ -29,4 +29,9 @@
   <div class="tabs-panel" id="tab-remote-census-configuration">
     <%= render "remote_census_configuration_tab" %>
   </div>
+
+  <div class="tabs-panel" id="tab-sdg-configuration">
+    <%= render "sdg_configuration_tab" %>
+  </div>
+
```

Comment:
For consistency with the way the code was till now, remove this blank line :pray:.